notebooks/en/index.md

```diff
@@ -7,6 +7,7 @@ applications and solving various machine learning tasks using open-source tools
 
 Check out the recently added notebooks:
 
+- [Signature-Aware Model Serving from MLflow with Ray Serve](mlflow_ray_serve)
```

Comment:
Let's do the following here so only 5 notebooks are shown with the most recent one on top:

```md
- Signature-Aware Model Serving from MLflow with Ray Serve
- Fine-tuning SmolVLM using direct preference optimization (DPO) with TRL on a consumer GPU
- Smol Multimodal RAG: Building with ColSmolVLM and SmolVLM on Colab’s Free-Tier GPU
- Fine-tuning SmolVLM with TRL on a consumer GPU
- Multimodal RAG with ColQwen2, Reranker, and Quantized VLMs on Consumer GPUs
```

Reply:
Done